toqito/state_opt/state_distinguishability.py

```diff
@@ -3,33 +3,38 @@
 import numpy as np
 import picos
 
-from toqito.matrix_ops import calculate_vector_matrix_dimension, vector_to_density_matrix
+from toqito.matrix_ops import calculate_vector_matrix_dimension, vector_to_density_matrix, vectors_to_gram_matrix
 from toqito.matrix_props import has_same_dimension
 
 
 def state_distinguishability(
     vectors: list[np.ndarray],
     probs: list[float] = None,
+    strategy: str = "min_error",
     solver: str = "cvxopt",
     primal_dual: str = "dual",
-) -> float:
+    **kwargs,
+) -> tuple[float, list[picos.HermitianVariable] | tuple[picos.HermitianVariable] | tuple[picos.RealVariable]]:
     r"""Compute probability of state distinguishability :cite:`Eldar_2003_SDPApproach`.
 
     The "quantum state distinguishability" problem involves a collection of :math:`n` quantum states
 
     .. math::
-        \rho = \{ \rho_0, \ldots, \rho_n \},
+        \rho = \{ \rho_1, \ldots, \rho_n \},
 
     as well as a list of corresponding probabilities
 
     .. math::
-        p = \{ p_0, \ldots, p_n \}.
+        p = \{ p_1, \ldots, p_n \}.
```

Comment:
Same here as the `rho_0` <-> `rho_1` comment.

Reply:
Resolving due to https://github.com/vprusso/toqito/pull/614#discussion_r1624748864